src/core/expr/fexpr.cc

```diff
@@ -442,6 +442,20 @@ DECLARE_METHOD(&PyFExpr::min)
     ->docs(dt::doc_FExpr_min);
 
 
+oobj PyFExpr::nth(const XArgs& args) {
+  auto nthFn = oobj::import("datatable", "nth", "skipna");
```

Comment:
I'm not sure what means imporing `skipna`. You only need to import `nth` from `datatable`.

Reply:
how do I pull in the `skipna` parameter into the `f` method?